### DISCUSSION TITLE
Override Windows 10 default system accent color

### DISCUSSION THREAD

**Discussion by DiagProf** [dotnet/maui#6304] [Q&A]
I'm looking for a way to override the default Windows10 accent color in my .Net Maui Windows app. For all controls where they appear. Could someone tell me what the right way is?

here is an example where the accent color comes through. You see it doesn't fit to a theme.

![accentColor](https://user-images.githubusercontent.com/56319388/164222488-41723580-497c-4c04-94f9-7ccd993d00d8.png)

and here where to set the accent color under Windows 10. More for info
![Windows10accentColor](https://user-images.githubusercontent.com/56319388/164221991-55179619-897f-45bf-8af2-430a8188c3f5.png)

**Comment by EORG-Dev** [comment]
I second this issue. I can't - for the life of me - find a solution to this. I cant even find any documentation to miss understand...
